### PR TITLE
Make create_table idempotent

### PR DESCRIPTION
**Pull request by ThHareau**
Makes `create_table` idempotent.

### CODE REVIEW

File: lib/safe-pg-migrations/plugins/idem_potent_statements.rb [hunk 1]
```diff
@@ -42,6 +42,15 @@ def add_foreign_key(from_table, to_table, **options)
       )
     end
 
+    def create_table(table_name, comment: nil, **options)
+      return super unless table_exists?(table_name)
+
+      SafePgMigrations.say(
+        "/!\\ Table '#{table_name}' already exists. Skipping statement.",
```
Comment (rpellerin) [inline lib/safe-pg-migrations/plugins/idem_potent_statements.rb:49]:
```suggestion
        "/!\ Table '#{table_name}' already exists. Skipping statement.",
```

**Review by rpellerin — APPROVED** [review]

File: test/safe_pg_migrations_test.rb [hunk 2]
```diff
@@ -193,6 +193,26 @@ def up
...
+          end
+        end
+      end.new
+
+    write_calls = record_calls(@migration, :write) { run_migration }.map(&:first)
+
+    assert_equal [
+      '== 8128 : migrating ===========================================================',
+      '-- create_table(:users)',
+      "   -> /!\\ Table 'users' already exists. Skipping statement.",
```
Comment (rpellerin) [inline test/safe_pg_migrations_test.rb:212]:
```suggestion
      "   -> /!\ Table 'users' already exists. Skipping statement.",
```

Comment (ThHareau) [reply test/safe_pg_migrations_test.rb:212]: image

It looks like backslash needs to be escaped. Am I missing something?

Comment (ThHareau) [reply test/safe_pg_migrations_test.rb:212]: Should I use single quote instead? And escape the quotes around `users`?

Comment (rpellerin) [reply test/safe_pg_migrations_test.rb:212]: don't know ^^ not blocking anyway

Comment (ThHareau) [reply test/safe_pg_migrations_test.rb:212]: OK, I'm keeping the `"\\"` as it's used in the rest of the code

**Pull request merged**